skeleton-es2016-aspnetcore/src/skeleton-navigation-es2016-vs/Views/Home/Index.cshtml

```diff
@@ -1,12 +0,0 @@
-﻿<div aurelia-app="main">
-    <div class="splash">
-        <div class="message">Aurelia Navigation Skeleton</div>
-        <i class="fa fa-spinner fa-spin"></i>
-    </div>
-
-    <script src="jspm_packages/system.js"></script>
-    <script src="config.js"></script>
-    <script>
-        System.import('aurelia-bootstrapper');
-    </script>
-</div>
```

Comment:
@mikekidder 
Even though not strictly needed (see above), I've reconsidered and since the project is an MVC project I will go ahead and add the view back in. I am going to be updating to the .RTM version of ASP.Net Core on the 27th so I'll include the view with those changes.